azure/v14.1.7/README.md

```diff
@@ -0,0 +1,45 @@
+# :zap: Giant Swarm Release v14.1.7 for Azure :zap:
+
+<< Add description here >>
```

Comment:
OK last change here (I guess something very similar to the announcement contents will do) and then we're good to go